### PR TITLE
Added support for using storage api for auth info

### PR DESCRIPTION
**Pull request by HenningNormann**
## Description
Storage is currently changing the storage database from Cosmos to PostgreSQL. This PR uses a new API to get auth info from storage rather than getting the data from the database without using a Storage API.

## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: test/IntegrationTests/appsettings.json [hunk 1]
```diff
@@ -26,11 +26,13 @@
   "GeneralSettings": {
     "OpenIdWellKnownEndpoint": "http://localhost:5040/authentication/api/v1/openid/",
     "BridgeApiEndpoint": "https://at22.altinn.cloud/sblbridge/authorization/api/",
+    "ApiStorageEndpoint": "http://localhost:5010/storage/api/v1/",
```
Comment (jonkjetiloye) [inline test/IntegrationTests/appsettings.json:29]: Environment variable for the new endpoint must also be added to values.yaml in altinn-studio-ops repo

**Review by jonkjetiloye — CHANGES_REQUESTED** [review]

**Comment by jonkjetiloye** [comment]
PR missing link to issue for the change

File: src/Authorization/appsettings.json [hunk 2]
```diff
@@ -36,10 +36,12 @@
     "RoleCacheTimeout": 5,
     "MainUnitCacheTimeout": 5,
     "KeyrolePartyIdsCacheTimeout": 5,
-    "PolicyCacheTimeout": 10
+    "PolicyCacheTimeout": 10,
+    "UseStorageApiForInstanceAuthInfo": true
```
[the remark this answers is not in the document]
Comment (HenningNormann) [reply src/Authorization/appsettings.json]: This was a mistake. It's changed now.

**Review by jonkjetiloye — APPROVED** [review]

File: test/IntegrationTests/appsettings.json (hunk 1 above)
Comment (HenningNormann) [reply test/IntegrationTests/appsettings.json:29]: Done in PR664

**Pull request merged**